plan/logical_plans.go

```diff
@@ -179,11 +179,12 @@ type LogicalAggregation struct {
 
 	AggFuncs     []expression.AggregationFunction
 	GroupByItems []expression.Expression
-
 	// groupByCols stores the columns that are group-by items.
 	groupByCols []*expression.Column
 
 	possibleProperties [][]*expression.Column
+	childCount         float64 // childCount is the child plan's count.
```

Comment:
maybe `inputCount` is a better name. childCount seems to indicate the number of children.